src/commands/verify.ts

```diff
@@ -0,0 +1,36 @@
+import { execSync } from "child_process"
+import { readFileSync } from "fs"
+import { existsSync } from "fs"
+import { writeFileSync } from "fs"
+import { resolve } from "path"
+import { PackageFormat } from "../api/PackageFormat"
```

Comment:
nit changing `api` to `config` fixes the build.

Reply:
```suggestion
import { PackageFormat } from "../config/PackageFormat"
```